### ISSUE TITLE
Get Each and every post for hashtag

### ISSUE THREAD

**Issue by FarzanRNobakht** [rarcega/instagram-scraper#211]
When you visit the explore page of instagram like

https://www.instagram.com/explore/tags/bookss/

we see that there is 1217 post in whole instagram with this tag
is it possible to get exactly ALL of these 1217 photos?

**Comment by ngSuraj** [comment]
@FarzanRNobakht No, We can't all the posts for a hashtag, only 80-90% of posts will be scrapped.

**Comment by FarzanRNobakht** [comment]
Are there any clues that which posts can be scrapped and  which cannot be scrapped?
